src/main/java/de/retest/persistence/migration/XmlTransformer.java

```diff
@@ -57,19 +55,13 @@ public final InputStream transform( final InputStream inputStream ) {
 				final XMLEvent nextEvent = eventReader.nextEvent();
 				convert( nextEvent, eventWriter );
 			}
-
 			eventReader.close();
 			eventWriter.flush();
 			eventWriter.close();
```

Comment:
It has nothing really to do with your PR, but could we use try-with-resources for the above close?